### DISCUSSION TITLE
Please add your very nice face recognition AI to also detect animals in addition to people

### DISCUSSION THREAD

**Discussion by eriklysoe** [immich-app/immich#10550] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be a very good future upgrade, many like myself have many pictures of animals. For us is several 100 pictures of our horses and cats

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Thank you. Duplicate of #7151
